package.json

```diff
@@ -29,18 +30,18 @@
     "analyze": "yarn ts-check && yarn flow-check",
     "flow-check": "npx flow-bin check-contents < src/index.js.flow",
     "ts-check": "npx tsc --noEmit",
-    "clean": "cd example && npx react-native-clean-project --keep-node-modules --remove-iOS-build --remove-iOS-pods --remove-android-build --clean-android-project --keep-brew --keep-pods && \\rm -fr ios/Pods",
-    "dev-sync": "yarn build && cp -r package.json *podspec lib windows android ios src jest example/node_modules/react-native-device-info/",
+    "clean": "cd FabricExample && npx react-native-clean-project --keep-node-modules --remove-iOS-build --remove-iOS-pods --remove-android-build --clean-android-project --keep-brew --keep-pods && \\rm -fr ios/Pods",
```

Comment:
So I understand `FabricExample` should be now the default option for testing?

Reply:
Yes, I think so unless we want to update the old example

Reply:
So maybe just rename it to `Example` and add some scripts to run it on old/new arch for CI?